specs/integration/config.js

```diff
@@ -0,0 +1,13 @@
+/* You can overwrite this config using the --config command-line option.
+See: https://github.com/MattiSG/Watai/wiki/Configuration
+E.g.: $ watai specs/integration/  --config '{"baseURL":"http://example.org"}'
+
+Note that you will have to set at least these config options
+in order to perform authenticated operations:
+
+$ --config '{"email":"david@larlet.fr","password":"testpassword","username":"David Larlet"}'
```

Comment:
Mmh… Not an exact answer to the specific problem you bring up, but you could be conceptually interested in what I do [here](https://github.com/sgmap/cout-embauche/blob/gh-pages/circle.yml#L20-L21).